installWSAGA.sh

```diff
@@ -0,0 +1,86 @@
+LINK='\033[1;34m' #blue
+YELLOW='\033[0;33m'
+NC='\033[0m' # No Color
+echo "updating..."
+sudo apt update
+sudo apt install unzip lzip git
+
+while ! find MicrosoftCorporationII.WindowsSubsystemForAndroid*.7z 1> /dev/null 2>&1
```

Comment:
Why is there 7z here?

Reply:
It's the file from https://store.rg-adguard.net

Reply:
store.rg-adguard.net tells me that the file extension is .msixbundle.
`MicrosoftCorporationII.WindowsSubsystemForAndroid_1.8.32822.0_neutral_~_8wekyb3d8bbwe.msixbundle`

Reply:
Oh, I might have renamed it locally. msixbundle is probably correct